apps/mobile-alphabet/android/app/src/main/res/values/styles.xml

```diff
@@ -4,6 +4,8 @@
     <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
         <!-- Customize your theme here. -->
         <item name="android:editTextBackground">@drawable/rn_edit_text_material</item>
+        <item name="android:navigationBarColor">#2044e0</item>
```

Comment:
Is this specific to TNG's build? If so, we might want to add a TODO to make it customizable.

Reply:
I changed this so it is semi-transparent now and the background color will match the windowBackground (startup background).